README.md

```diff
@@ -1 +1,25 @@
 # Node.js userland migrations
+
+This repository contains codemodes (automated migrations) for "userland" code. These are intended to facilitate adopting new features and upgrading source-code affected by breaking changes.
+
+[![Formatted with Biome](https://img.shields.io/badge/Formatted_with-Biome-60a5fa?style=flat&logo=biome)](https://biomejs.dev/) [![Linted with Biome](https://img.shields.io/badge/Linted_with-Biome-60a5fa?style=flat&logo=biome)](https://biomejs.dev) [![Checked with Biome](https://img.shields.io/badge/Checked_with-Biome-60a5fa?style=flat&logo=biome)](https://biomejs.dev)
```

Comment:
nit: i am not a big fan of these. they are bumperstickers, and are often not accessible to screenreaders. YMMV

Reply:
I think these don't add much value—they seem more marketing for biome. @AugustinMauroy you added them. do you feel strongly?

Reply:
I just add it like that any strong opinion.

Reply:
Sorry, I don't understand—you're saying you _do_ feel strongly?

Reply:
I wanted to say that I just added it like that. So it doesn't matter to me whether it's there or not.